src/controllers/contacts.js

```diff
@@ -1,7 +1,18 @@
+const jwt = require("jsonwebtoken");
+
 const getContacts = require("../model/queries/getContacts");
 
 const contacts = (req, res) => {
-  getContacts().then(response => {
+
+  let email = '';
+
+  const token = req.headers.cookie.split("=")[1];
+  jwt.verify(token, process.env.SECRET, function (err, decoded) {
+    email = decoded.user_email;
+    console.log('this is decoded,>>>>', decoded);
+  });
+
+  getContacts(email).then(response => {
```

Comment:
This pattern is likely to cause problems. `getContacts` should be inside the `jwt.verify` callback

Reply:
Ok thanks Sam. We will look into that now.

Reply:
Thanks, for pointing that out. We'll get it sorted in a bit. 😄